legal-api/src/legal_api/models/business.py

```diff
@@ -132,6 +132,19 @@ class LegalTypes(str, Enum):
         LegalTypes.ULC_CONTINUE_IN.value,
     ]
 
+    ELIGIBLE_TYPES: Final = [
+        LegalTypes.COMP.value,
+        LegalTypes.BC_ULC_COMPANY.value,
+        LegalTypes.BC_CCC.value,
+        LegalTypes.BCOMP.value,
+        LegalTypes.CONTINUE_IN.value,
+        LegalTypes.ULC_CONTINUE_IN.value,
+        LegalTypes.CCC_CONTINUE_IN.value,
+        LegalTypes.BCOMP_CONTINUE_IN.value,
+        LegalTypes.EXTRA_PRO_A.value,
+        LegalTypes.LIMITED_CO.value
+    ]
```

Comment:
you can use [CORPS](https://github.com/bcgov/lear/blob/acdc05ccff4e56131790e1cc650b42df95babfc9/legal-api/src/legal_api/models/business.py#L124-L133) in the business model instead.

Reply:
Actually, `CORPS` is missing `LegalTypes.EXTRA_PRO_A.value` and `LegalTypes.LIMITED_CO.value`.

I think you should create an `ELIGIBLE_TYPES` list in the involuntary dissolutions service class.  Don't think it belongs in the business model.

Reply:
Updated